### PR TITLE
page model detail page

### PR DESCRIPTION
**Pull request by alessioscalici**

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: src/state/page-models/actions.js [hunk 1]
```diff
@@ -143,3 +159,27 @@ export const createPageModel = pageModel => (dispatch, getState) => {
...
+            dispatch(setPage(json.metaData));
+            resolve();
+          } else {
+            dispatch(addErrors(json.errors.map(err => err.message)));
+            dispatch(toggleLoading('references'));
+            resolve();
+          }
+        });
+      });
+    });
```
Comment (fedelocci) [inline src/state/page-models/actions.js]: you can optimize the function like this: 

const fetchCurrentReference = (getApiCall, setActionCreator) =>
  (page = { page: 1, pageSize: 10 }) => (dispatch, getState) =>
    new Promise((resolve) => {
      const { pageModelCode } = getParams(getState());
      dispatch(toggleLoading('references'));
      getApiCall(pageModelCode, page).then((response) => {
        response.json().then((json) => {
          if (response.ok) {
            dispatch(setActionCreator(json.payload));
            dispatch(setPage(json.metaData));
          } else {
            dispatch(addErrors(json.errors.map(err => err.message)));
          }
            dispatch(toggleLoading('references'));
            resolve();
        });
      });
    });

File: test/state/page-models/actions.test.js [hunk 2]
```diff
@@ -80,6 +81,25 @@ describe('state/page-models/actions', () => {
...
+    it('is FSA compliant', () => {
+      expect(isFSA(action)).toBe(true);
+    });
+
+    it('is of type SET_SELECTED_PAGE_MODEL_PAGE_REFS', () => {
+      expect(action.type).toBe(SET_SELECTED_PAGE_MODEL_PAGE_REFS);
+    });
+
+    it('defines the "references" property', () => {
+      expect(action.payload.references).toEqual(PAGE_REFS);
```
Comment (fedelocci) [inline test/state/page-models/actions.test.js]: it's better toHaveProperty ?

File: test/ui/app/App.test.js [hunk 3]
```diff
@@ -222,6 +224,11 @@ describe('App', () => {
     expect(component.contains(<PageModelEditPage />)).toEqual(true);
   });
 
+  it('route to page model detail page', () => {
+    const component = shallow(<App route={ROUTE_PAGE_MODEL_DETAIL} username="admin" />);
+    expect(component.contains(<PageModelDetailPageContainer />)).toEqual(true);
```
Comment (fedelocci) [inline test/ui/app/App.test.js]: toBe

File: test/ui/app/App.test.js [hunk 4]
```diff
@@ -222,6 +224,11 @@ describe('App', () => {
...
+  });
+
   it('default route', () => {
     const component = shallow(<App route="test" username="admin" />);
     expect(component.contains(<NotFoundPage />)).toEqual(true);
```
Comment (fedelocci) [inline test/ui/app/App.test.js]: toBe

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Pull request merged**